include/envoy/config/typed_metadata.h

```diff
@@ -65,6 +66,9 @@ class TypedMetadataFactory : public UntypedFactory {
   virtual std::unique_ptr<const TypedMetadata::Object>
   parse(const ProtobufWkt::Struct& data) const PURE;
 
+  virtual std::unique_ptr<const TypedMetadata::Object>
```

Comment:
It might be better to have the comment for the two `parse` methods separate (one for Struct and one for Any). When looking at the function, one immediately sees the comment, and it also allows IDEs to show the function comment.

Reply:
Yes, make sense. Done!